src/game_api/memory.hpp

```diff
@@ -7,6 +7,80 @@
 #include "search.hpp"
 
 using namespace std::string_literals;
+class ExecutableMemory

```

Comment:
Is there a good reason to move this? If not I'd prefer to put it back to where it was to reduce the size of the diff.

Reply:
needed to have this above the functions as they use get function from them, and functions needs to be in the `namespace
{` thing... for whatever reason, otherwise it just throws tons of errors
i actually hate this file, i prefer to have the usual header/source as two separate files, but i didn't know if this was made this way on purpose or not